generic_throttle/src/generic_throttle_node.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python
```

Comment:
I think the generic_throttle_node.py should node be within the src folder - which is reserved for the python module...
put it under generic_throttle/scripts